### ISSUE TITLE
Two more flake8-boolean-trap FBT003 false positives

### ISSUE THREAD

**Issue by rhkleijn** [charliermarsh/ruff#1336]
I highly appreciate  your amazing work on ruff!

I noticed your recent PR #1333 . In my code base I encountered two more flake8-boolean-trap FBT003 false positives for `getattr` and for the `index` method of `list`, `tuple` and other Sequences. E.g.

```python
getattr(someobj, attrname, False)  # return False if `attrname` is not found on `someobj`
mylist.index(True)  # find index of first occurrence of True
```

It would be great to get rid of some `# noqa: FBT003` comments for these cases.

**Comment by charliermarsh** [comment]
Will add!
